Makefile

```diff
@@ -7,3 +10,11 @@ protobuf:
 	cd Tests/Protobuf; \
```

Comment:
Can you update the `protobuf:` task to use `PROTO_DIR` & friend, similar to your new addition that uses Docker container to generate code from the protos?